data_pipeline/README.md

```diff
@@ -0,0 +1,158 @@
+# Data Pipeline Workflow
+
+This is an example of a data pipeline workflow. The pipeline is triggered by a new GPX file on an S3 bucket.
+The pipeline code will parse the GPX file and insert it into a database.
+
+The event flow is:
+
+- S3 sends a new item notification to an SNS topic
+- SNS sends a notification to the AutoKitteh HTTP trigger
+- The workflow extracts the bucket and file name from the S3 event
+- The workflow reads the file from S3
+- The workflow parses the GPX file and inserts the data into a SQLite database
+
+## Setting Up
+
+### Starting `ak`
+
+Start `ak`, you can use development mode
+
+```
+ak up --mode dev
+```
+
+You'll need to expose `ak` web API to the outside world in order to get events from S3.
+In this example, we'll use [ngrok](https://ngrok.com/)
+
+```
+ngrok http 9980
+```
+
+You'll see a screen with a line like:
+
+```
+Forwarding                    https://e7c499cae8d9.ngrok.app -> http://localhost:9980
```

Comment:
Extreme nit: even though the spaces are accurate, they're not serving a useful purpose here.

```suggestion
Forwarding https://e7c499cae8d9.ngrok.app -> http://localhost:9980
```

Reply:
I prefer to show output that is as similar to the real one.